### ISSUE TITLE
Qustion about SFT

### ISSUE THREAD

**Issue by ccccj** [mit-han-lab/llm-awq#112]
Hello, I have a model of llama that has been fine-tuned using my own dataset. Is it possible for me to compress the fine-tuned llama model and will the accuracy drop due to my fine-tuning?

**Comment by ccccj** [comment]
I mean, is it possible that I perform AWQ search again and will get better performance?

**Comment by Hongbosherlock** [comment]
> I mean, is it possible that I perform AWQ search again and will get better performance?

hi, I have similar issue. Do you have any progress?

**Comment by ccccj** [comment]
> Hello, I have a model of llama that has been fine-tuned using my own dataset. Is it possible for me to compress the fine-tuned llama model and will the accuracy drop due to my fine-tuning?

No, I used another method of compressing the model(SpQR), so I didn't use this one

**Comment by Hongbosherlock** [comment]
> > Hello, I have a model of llama that has been fine-tuned using my own dataset. Is it possible for me to compress the fine-tuned llama model and will the accuracy drop due to my fine-tuning?
> 
> No, I used another method of compressing the model(SpQR), so I didn't use this one

Did SpQR achieve good results ?  btw, I heard that SpQR doesn't have the corresponding CUDA kernel implemented.

**Comment by ccccj** [comment]
> > > Hello, I have a model of llama that has been fine-tuned using my own dataset. Is it possible for me to compress the fine-tuned llama model and will the accuracy drop due to my fine-tuning?
> > 
> > 
> > No, I used another method of compressing the model(SpQR), so I didn't use this one
> 
> Did SpQR achieve good results ? btw, I heard that SpQR doesn't have the corresponding CUDA kernel implemented.

spqr looks ok from the score (my application scenario doesn't require the implementation of a cuda kernel). Maybe we can communicate by e-mail or WeChat.

**Comment by Hongbosherlock** [comment]
> > > > Hello, I have a model of llama that has been fine-tuned using my own dataset. Is it possible for me to compress the fine-tuned llama model and will the accuracy drop due to my fine-tuning?
> > > 
> > > 
> > > No, I used another method of compressing the model(SpQR), so I didn't use this one
> > 
> > 
> > Did SpQR achieve good results ? btw, I heard that SpQR doesn't have the corresponding CUDA kernel implemented.
> 
> spqr looks ok from the score (my application scenario doesn't require the implementation of a cuda kernel). Maybe we can communicate by e-mail or WeChat.

Sure, My WeChat  is all lowercase words of my github nickname.
